package/src/components/MessageList/MessageList.tsx

```diff
@@ -525,7 +525,7 @@ const MessageListWithContext = <
     if (threadList || hasNoMoreRecentMessagesToLoad) {
       scrollToBottomIfNeeded();
     } else {
-      setScrollToBottomButtonVisible(true);
+      setScrollToBottomButtonVisible(false);
     }
```

Comment:
this seems like a big change...

have you also tested with jump to message? its in sample-app mention screen

Reply:
This works fine.